pyproject.toml

```diff
@@ -55,6 +55,7 @@ dependencies = [
     "mysql-connector-python >= 8.0.29",
     "python-redmine >= 2.3.0",
     "requests >= 2.28.0",
+    "spython >= 0.3.13",
```

Comment:
Should we create a labelled dependency instead for `datasets` or running containers? Just an idea.

Reply:
I suppose so yes. Though datasets will be very integrated in the not so distant future. So do we want to separate it out from base deps ?

Reply:
Dependencies on third-party tools such as binaries have to be included in the installation instructions rather than in the `pyproject.toml`. For now let's leave this as it is.